libs/components/modals/src/lib/modules/modal/modal.component.scss

```diff
@@ -188,6 +189,23 @@
     padding-top: initial;
     width: 100%;
   }
+
+  .sky-modal-viewkeeper {
+    .sky-modal-header {
+      box-shadow: none !important;
+    }
+
+    .sky-modal-content
```

Comment:
This seems like it would be better off in `modal-content.component.scss`, which has view encapsulation disabled and wouldn't require `::ng-deep`.

Reply:
The alternatives weren't better. Sticking with `::ng-deep`.